README.md

```diff
@@ -41,12 +41,14 @@ Steps:
     1. Make a copy of `DEV-reporting.publish.xml` and name it `DEV-USER-reporting.publish.xml`. This file will be ignored by Git.
     1. Make a copy of `DEV-reporting-DELETE.publish.xml` and name it `DEV-USER-reporting-DELETE.publish.xml`. This file will be ignored by Git.
 1. If the instance of SQL Server that you are using is not at `localhost` then update the `Data Source` in the `TargetConnectionString`.
+1. Run the NTBS app, so that the ntbsDev and ntbsAudit databases are migrated (as some reporting views depend on these).
```

Comment:
You should have done this already in point 1 where you set up the NTBS application. I think maybe make this a subpoint of point 1, like "make sure that you have run the NTBS app, so that the application and audit databases are migrated"